js/index.js

```diff
@@ -0,0 +1,79 @@
+const countriesGrid = document.querySelector(".countries-grid");
+const searchInput = document.querySelector(".search-input");
+const dropdownHeader = document.querySelector(".dropdown-header");
+const dropdownBody = document.querySelector(".dropdown-body");
+const dropdownItems = document.querySelectorAll(".dropdown-body ul li");
+
+// Fetch data from CountriesData.json
+fetch("./CountriesData.json")
+  .then((response) => response.json())
```

Comment:
Why not extract his into a function and call if `getCountriesDate()` 
And call it instead of putting a comment